agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java

```diff
@@ -242,7 +242,7 @@ private void exportRemoteDependency(SpanData span, boolean inProc) {
 
     // set dependency-specific properties
     data.setId(span.getSpanId());
-    data.setName(getTelemetryName(span));
+    data.setName(span.getName());
```

Comment:
It seems to be just "operation name" where we want special AI-specific naming, I think ok to map directly from span name here (cc: @vishweshbankwar)